app/uk/gov/hmrc/cgtpropertydisposals/models/ContactDetails.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposals.models
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class ContactDetails(
```

Comment:
if we're going to start using this for the update request, we should be using it as well use it in the create subscription request and display subscription response for consistency - probably not now though since the FE will need to change as well. Unless you want to do that now?

Reply:
Yeah, let's leave it for now. The next tickets are to work on the FE so could look at them then and get that done.